test/built-ins/Array/prototype/groupToMap/map-instance.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.groupToMap
+description: Array.prototype.groupToMap returns a null prototype object
```

Comment:
```suggestion
description: Array.prototype.groupToMap returns a Map instance
```
Copy-paste error? :sweat_smile: